support-frontend/app/controllers/Application.scala

```diff
@@ -129,9 +129,12 @@ class Application(
 
     val js = elementForStage(RefPath("contributionsLandingPage.js"))
 
+    val mainElement = assets.getSsrCacheContentsAsHtml(s"contributions-landing-page-$countryCode","contributions-landing.html")
```

Comment:
unrelated to your PR but we should probably ditch the country code in the main element id, it's unnecessary and a potential source of bugs. in fact even better would be to have a universal root element id across all pages